code/ObjFileMtlImporter.cpp

```diff
@@ -247,13 +247,16 @@ void ObjFileMtlImporter::getColorRGBA( aiColor3D *pColor )
     m_DataIt = getFloat<DataArrayIt>( m_DataIt, m_DataItEnd, r );
     pColor->r = r;
 
+    skipSpaces( m_DataIt, m_DataIt);
+
     // we have to check if color is default 0 with only one token
     if( !IsLineEnd( *m_DataIt ) ) {
         m_DataIt = getFloat<DataArrayIt>( m_DataIt, m_DataItEnd, g );
         m_DataIt = getFloat<DataArrayIt>( m_DataIt, m_DataItEnd, b );
+    } else {
+        pColor->g = r;
+        pColor->b = r;
```

Comment:
r?

Reply:
the red component value just read before

Reply:
I was just asking to be sure it wasn't a mistake to assign `red` to `green` and `blue`